lib/tapioca/generator.rb

```diff
@@ -316,6 +358,7 @@ def add(filename)
 
     sig { params(filename: Pathname).void }
     def remove(filename)
+      return unless filename.exist?
```

Comment:
This is a nice touch in case user passes in bad input ie. not a fully qualified name 👍